corehq/apps/app_manager/tests/test_translations.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf8
+from lxml import etree
+from django.test import SimpleTestCase
+from corehq.apps.app_manager.translations import escape_output_value
+
+
+class AppManagerTranslationsTest(SimpleTestCase):
+
+    def test_escape_output_value(self):
+        test_cases = [
```

Comment:
would be nice to create a comprehensive set of tests that we think we care about. I wasn't sure I had enough context.